packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "main": "out/src",
   "dependencies": {
-    "@salesforce/core": "3.34.7",
+    "@salesforce/core": "^3.34.8",
```

Comment:
In all packages we've had to update core to "^3.34.8" so that the [same version that is installed by @salesforce/source-deploy-retrieve](https://github.com/forcedotcom/source-deploy-retrieve/blob/main/package.json#L28) is also installed for all of the VSCEs.